include/mtrie.h

```diff
@@ -0,0 +1,40 @@
+// Sprint 2 Mock Trie Implementation
+// Ruolin Zheng
+
+
+// minimal C implementation of a mock trie
+
+#ifndef __TRIE_H__
+#define __TRIE_H__
+ 
+// Alphabet size (# of symbols)
+#define ALPHABET_SIZE (26)
+ 
+typedef struct trie_t
+{
+	char *value;
+    struct trie_t *children[ALPHABET_SIZE];
+} trie_t;
+
+
+// Returns 1 if key presents in trie, else 0
```

Comment:
These are not acceptable header comments. Please include full header comments in the standardized style the group decides on.